src/components/Header/Header.jsx

```diff
@@ -1,11 +1,25 @@
-import React from "react";
-import './Header.scss'
+import React, { useState } from "react";
+import "./Header.scss";
 
-const Header = ({title}) =>
-    <header className="c-site-header">
-        <h1 className="c-site-header__title o-type__invisible">{title}</h1>
-        {/* look up component */}
-    </header>
+const Header = ({getNewLocation}) => {
+  const [city, setCity] = useState("");
+console.log({ city });
 
+  return (
+    <header className="header">
+      <input
+        className="Location"
+        
+        placeholder="Type in a city name"
```

Comment:
For accessibility, inputs should have labels. If the label isn't visible in the design, you can hide it except for screen readers with a .visually-hidden class (ref - https://css-tricks.com/inclusively-hidden/)